ProcessLib/ProcessVariable.cpp

```diff
@@ -12,19 +12,19 @@
 #include <utility>
 #include <logog/include/logog.hpp>
 
-#include "GeoLib/GEOObjects.h"
 #include "MeshLib/Mesh.h"
 #include "ProcessLib/Utils/ProcessUtils.h"
 
 namespace ProcessLib
 {
 ProcessVariable::ProcessVariable(
-    BaseLib::ConfigTree const& config, MeshLib::Mesh& mesh,
-    GeoLib::GEOObjects const& geometries,
+    BaseLib::ConfigTree const& config,
+    std::vector<MeshLib::Mesh*> const& meshes,
     std::vector<std::unique_ptr<ParameterBase>> const& parameters)
     :  //! \ogs_file_param{prj__process_variables__process_variable__name}
       _name(config.getConfigParameter<std::string>("name")),
-      _mesh(mesh),
+      _mesh(*meshes[0]),  // Using the first mesh as the main mesh.
+                          // TODO (naumov) potentially extend to named meshes.
```

Comment:
For the time being:
Does the name mesh have a specific name? E.g. an empty one? Maybe then check that the name matches.

Reply:
The mesh's name constructed from a vtk mesh reader is the base name w/o extension of the file name.
But the file name is not known here... Discussion?